libraries/Microsoft.Bot.Builder.Ai/QnAMakerMiddlewareOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Microsoft.Bot.Builder.Ai
+{
+    /// <summary>
+    /// Options to alter the default behaviour of the QnA Maker Middleware
+    /// extending the QnAMakerOptions used on the QnAMaker client
+    /// </summary>
+    public class QnAMakerMiddlewareOptions : QnAMakerOptions
```

Comment:
The code you have in the ctor for the Middlware is not taking advantage of this "is a" relationship.